pkg/reconciler/pipelinerun/resources/pipelineref.go

```diff
@@ -156,21 +156,28 @@ func readRuntimeObjectAsPipeline(ctx context.Context, namespace string, obj runt
 		obj.ObjectMeta.OwnerReferences = nil
 		// Verify the Pipeline once we fetch from the remote resolution, mutating, validation and conversion of the pipeline should happen after the verification, since signatures are based on the remote pipeline contents
 		vr := trustedresources.VerifyResource(ctx, obj, k8s, refSource, verificationPolicies)
-		// Issue a dry-run request to create the remote Pipeline, so that it can undergo validation from validating admission webhooks
-		// without actually creating the Pipeline on the cluster.
-		if err := apiserver.DryRunValidate(ctx, namespace, obj, tekton); err != nil {
+		// Issue a dry-run request to create the remote Task, so that it can undergo validation from validating admission webhooks
```

Comment:
NIT: was the original comment valid?

`Issue a dry-run request to create the remote Pipeline`

Reply:
Yes. copy-paste error 😅 Fixing.